cmake/application.cmake

```diff
@@ -2,10 +2,45 @@ include(FetchContent)
 
 set(FETCHCONTENT_QUIET FALSE)
 
-FetchContent_Declare(
-  application
-  DOWNLOAD_EXTRACT_TIMESTAMP TRUE
-  URL https://github.com/chengcli/application/archive/refs/tags/v0.7.tar.gz)
+set(patch_command
+    git apply ${CMAKE_CURRENT_SOURCE_DIR}/patches/19.calculate_fluxes.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/21.new_blockdt.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/23.meshblock.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/24.time_integrator.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/25.constant_acc.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/26.scalars_flux.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/26.adiabatic_hydro.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/27.coordinates.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/28.bvals_cc.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/29.mesh.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/30.bvals_base.patch
+    ${CMAKE_CURRENT_SOURCE_DIR}/patches/31.task_list.patch)
```

Comment:
The statements above are incorrect. These patch files only apply to athena.cmake, not here in application.cmake